src/libs/duplicate.c

```diff
@@ -353,11 +353,11 @@ void gui_init(dt_lib_module_t *self)
   GtkWidget *bt = gtk_label_new(_("existing duplicates"));
   gtk_box_pack_start(GTK_BOX(hb), bt, FALSE, FALSE, 0);
   bt = dtgtk_button_new(dtgtk_cairo_paint_plusminus, CPF_ACTIVE | CPF_STYLE_FLAT | CPF_DO_NOT_USE_BORDER, NULL);
-  g_object_set(G_OBJECT(bt), "tooltip-text", _("create a 'virgin' duplicate of the image without any developpement"), (char *)NULL);
+  g_object_set(G_OBJECT(bt), "tooltip-text", _("duplicate image without its history"), (char *)NULL);
```

Comment:
What about:
"duplicate image without its history" -> "duplicate image with pristine history stack"

Reply:
That sounds good to me. I'll update this...